stellar_sdk/keypair.py

```diff
@@ -278,12 +278,12 @@ def generate_shamir_mnemonic_phrases(
             message = "shamir_mnemonic must be installed to use method `generate_shamir_mnemonic_phrases`."
             raise ModuleNotFoundError(message) from exc
 
-        secrets = Keypair.random().secret.encode()
+        bip39_seed = os.urandom(128)
```

Comment:
Can you confirm if this should be renamed to bip39 seed? I think it can use bip 39 seed, but it doesn't necessarily have to be bip 39 seed.